README.md

```diff
@@ -1,25 +1,68 @@
-# DeepCell Label: Data Curation Tools for DeepCell.
-[![Actions Status](https://github.com/vanvalenlab/deepcell-label/workflows/browser/badge.svg)](https://github.com/vanvalenlab/deepcell-label/actions)
+# Browser-Based Application of DeepCell Label for Data Curation
 
-DeepCell Label is a segmentation and tracking tool used for human-in-the-loop data curation. It displays lineage data along with raw and annotated images. The output files prepare this information as training data for DeepCell.
+This is an version of DeepCell Label that runs on a browser. The browser-based DeepCell Label application can be run locally or deployed to AWS Elastic Beanstalk.
+
+Many key and mouse operations are the same between desktop and browser versions, but these versions are not guaranteed to share the same features. See the "Controls" section for an up-to-date list of features.
```

Comment:
I don't think we should mention the desktop version here. Instead, "See the "Controls" section for an up-to-date list of features." should be enough